### DISCUSSION TITLE
Can't edit components in circuit box in sub sditor

### DISCUSSION THREAD

**Discussion by G-M-twostay** [FakeFishGames/Barotrauma#14068] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After setting "allow edit in game" to false for components that are in a circuit box, once you close the edit panel, you can't reopen it  to edit the values again.

### Reproduction steps

1. open sub editor.
2. place a circuit box and place any component in it(I have tested mem component, regex component).
3. click the placed component, in the property panel, uncheck "allow edit in game".
4. click somewhere else or do something to close the property panel.
5. select the component again and find out that the property panel won't open.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### -

Sub Editor

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/f6b0c0357c373e9d2beddd4ee019ea8823185d35 (commit in our private repo)

**Comment by Regalis11** [comment]
Merged the fix, coming up in the next unstable build.
